src/utils/pgwire/src/pg_server.rs

```diff
@@ -126,6 +127,11 @@ impl UserAuthenticator {
             UserAuthenticator::Md5WithSalt {
                 encrypted_password, ..
             } => encrypted_password == password,
+            UserAuthenticator::OAuth => {
+                // TODO: OAuth authentication happens here.
+                tracing::info!("OAuth authenticator gets: {}", String::from_utf8_lossy(password));
```

Comment:
Better to make it `debug` when ready to avoid log sensitive data, refer: #12673.

Reply:
It will be removed later. For now it is used to check the token is passed correctly.